marketplace_builder/views/pages/how-platformos-works/platformos-workflow/marketplace-kit.liquid

```diff
@@ -0,0 +1,181 @@
+|--------------	|-------------------------------------------	|-----------------------------------------	|
+| data export  	| `marketplace-kit data export <environment>` 	|  <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit data export production</code></pre> 	|
+| data import  	| `marketplace-kit data import <environment>` 	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit data import production</code></pre>  	|
+| data clean   	| `marketplace-kit data clean <environment>`  	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit data clean staging</code></pre>      	|
+
+
+#### deploy 
+Deploy code to environment. It is recommended to first deploy to staging, test, and only then trigger deploy to production as desribed in the recommended [Development Workflow](/how-platformos-works/platformos-workflow/development-workflow). 
+
+Effectively, deploy creates a zip file that contains all your files and sends it to the API. It is then processed in the background. Each zip file is stored by us, to allow you to roll back in case something goes wrong. 
+
+You can override changes made in the Instance admin by using the force flag. If you see a `locked_by_admin` error and you still want to deploy, use the `-f` flag.
+
+| Command                	| Usage                                   	| Example                           	|
+|------------------------	|-----------------------------------------	|-----------------------------------	|
+| deploy                 	| `marketplace-kit deploy <environment>`    	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit deploy staging</code></pre>     	|
+| deploy with force flag 	| `marketplace-kit deploy <environment> -f` 	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit deploy staging -f</code></pre> 	|
+
+**Related topics**: 
+
+* [Deploying and Syncing Changes](/get-started/deploying-syncing-changes)
+
+
+#### env
+Manage environments. 
+
+To add your environment to a config file, run the `env add` command, and authenticate with your Partner Portal credentials. Configuration for environments is saved in the `.marketplace-kit` file. 
+
+
+| Command                	| Usage                                   	| Example                           	|
+|------------------------	|-----------------------------------------	|-----------------------------------	|
+| env add                 	| `marketplace-kit env add <environment> --email <your email> --url <your marketplace url>`    	| <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">marketplace-kit env add staging --email myemail@example.com --url https://example.com</code></pre>     	|
```

Comment:
Looking at how it looks in the table i would go with simple backticks instead of html that inserts command line styling.

https://diana-documentation.staging.oregon.platform-os.com/how-platformos-works/platformos-workflow/marketplace-kit

![image](https://user-images.githubusercontent.com/546845/51683315-dbfad280-1fe9-11e9-91c6-d1b1e1f7a2a3.png)

(it has to look much worse on linux/windows where scrollbars are visible)

Reply:
Thanks, replaced all <pre ...> with backtick.